app/views/declaration/commodityMeasure/commodity_measure.scala.html

```diff
@@ -38,7 +38,7 @@
         appConfig: AppConfig
 )
 
-@(mode: Mode, itemId: String, form: Form[CommodityMeasure])(implicit request: JourneyRequest[_], messages: Messages)
+@(mode: Mode, itemId: String, form: Form[CommodityMeasure], commodityCode: Option[String])(implicit request: JourneyRequest[_], messages: Messages)
```

Comment:
Same here. Instead of passing `commodityCode: Option[String]` you could have something like..
```
request.cacheModel.getCommodityCodeOfItem(itemId)
```
btw not a big deal, not need to change it if you prefer to pass the argument.